spec.html

```diff
@@ -47041,6 +47168,11 @@ <h1>Static Semantics: Early Errors</h1>
         </ul>
       </emu-annex>
 
+      <emu-annex id="sec-thenumberofleftcapturingparens-annexb">
+        <h1>Static Semantics: TheNumberOfLeftCapturingParensIn and TheNumberOfLeftCapturingParensToTheLeftOf</h1>
+        <p>In the definitions of TheNumberOfLeftCapturingParensIn and TheNumberOfLeftCapturingParensToTheLeftOf, references to &ldquo;<emu-grammar>Atom :: `(` GroupSpecifier Disjunction `)`</emu-grammar> &rdquo; are to be interpreted as meaning &ldquo;<emu-grammar>Atom :: `(` GroupSpecifier Disjunction `)`</emu-grammar> &rdquo; or &ldquo;<emu-grammar>ExtendedAtom :: `(` Disjunction `)`</emu-grammar> &rdquo;.</p>
```

Comment:
Those operations should have forward amendment notices of the sort that appears in [IsCharacterClass](https://tc39.es/ecma262/multipage/text-processing.html#sec-patterns-static-semantics-is-character-class):
> ```html
> <emu-note>
>   <p>This section is amended in <emu-xref href="#sec-thenumberofleftcapturingparens-annexb"></emu-xref>.</p>
> </emu-note>
> ```

Reply:
I've added those in #2625 also.